tests/foreman/api/test_repository.py

```diff
@@ -2549,3 +2549,37 @@ def clean_repo():
             synced_repo = repo.read()
             assert synced_repo.content_counts['docker_manifest'] >= 1
             assert synced_repo.content_counts['docker_tag'] == 1
+
+
+@pytest.mark.tier2
+def test_positive_syncable_yum_format_repo_import(target_sat, module_org):
+    """Verify that you can import syncable yum format repositories
+
+    :id: dc6422c3-1ea1-4856-ab64-7c66e6043300
+
+    :steps:
+        1. Execute hammer command
+            hammer content-import repository --organization='' --path=''
+        2. Assert job is successful
+        3. Verify repository is synced and contains rpms
+
+    :expectedresults: The syncable repository sync correctly and all content is imported
+
+    :customerscenario: true
+
+    :BZ: 2128894
+    """
+    target_sat.execute('mkdir /var/lib/pulp/imports/syncable-repo')
+    target_sat.execute(
+        'wget -r -np -nH --cut-dirs=100 /var/lib/pulp/imports/syncable-repo'
+        f' {settings.robottelo.REPOS_HOSTING_URL}/exported_repo/'
+    )
```

Comment:
Would you create a common helper under hosts classes/its mixins for downloading repo along with its rpms/contents so that it could be used anywhere in robottelo in this or other components ?

Not a blocker for your PR